### ISSUE TITLE
Generate stats as part of each release

### ISSUE THREAD

**Issue by deepakunni3** [Knowledge-Graph-Hub/kg-covid-19#122]
Generate stats on the merged KG as part of each release.

At minimum, for node types:
```
105     biolink:Protein
112     biolink:Gene
227     biolink:NamedThing
283     biolink:Disease
1196    biolink:CellularComponent
2718    biolink:MolecularActivity
3254    biolink:CellularComponent
3541    biolink:OntologyClass
4392    biolink:BiologicalProcess
4655    biolink:AnatomicalEntity
9478    biolink:MolecularActivity
17981   biolink:Protein
20628   biolink:Gene
22887   biolink:ChemicalSubstance
26388   biolink:BiologicalProcess
32126   biolink:Drug
51302   biolink:Publication
67975   biolink:OntologyClass
```

Similarly for edge types.

These stats can be provided as markdown (text) along side the merged graph TSV.

**Comment by justaddcoffee** [comment]
Here's Laurent-Phillip's beautiful stats page we could mimic:
http://geneontology.org/stats.html

<img width="1330" alt="Screen Shot 2020-05-05 at 2 22 52 PM" src="https://user-images.githubusercontent.com/150311/81117430-ec337a00-8edb-11ea-8150-dbb8ead7f15b.png">

**Comment by justaddcoffee** [comment]
I think we've addressed this with #163
